manifests/database/postgresql.pp

```diff
@@ -1,23 +1,29 @@
 # Candlepin Database Setup using Postgreql
-class candlepin::database::postgresql{
+class candlepin::database::postgresql(
 
-  $db_dialect = 'org.hibernate.dialect.PostgreSQLDialect'
-  $db_quartz_dialect = 'org.quartz.impl.jdbcjobstore.PostgreSQLDelegate'
-  $db_driver = 'org.postgresql.Driver'
-  $db_type = $::candlepin::db_type
-  $db_host = $::candlepin::db_host
-  $db_port = pick($::candlepin::db_port, 5432)
-  $db_name = $::candlepin::db_name
-  $db_user = $::candlepin::db_user
-  $db_password = $::candlepin::db_password
-  $enable_hbm2ddl_validate = $::candlepin::enable_hbm2ddl_validate
+  $candlepin_conf_file     = $::candlepin::candlepin_conf_file,
+  $db_dialect              = 'org.hibernate.dialect.PostgreSQLDialect',
+  $db_quartz_dialect       = 'org.quartz.impl.jdbcjobstore.PostgreSQLDelegate',
+  $db_driver               = 'org.postgresql.Driver',
+  $manage_db               = $::candlepin::manage_db,
+  $init_db                 = $::candlepin::init_db,
+  $db_type                 = $::candlepin::db_type,
+  $db_host                 = $::candlepin::db_host,
+  $db_port                 = pick($::candlepin::db_port, 5432),
```

Comment:
Fair enough. There is currently a failure in the pipeline I'd like to investigate prior to merging this to prevent potentially compounding the effect.